libs/eo/transfers/src/lib/eo-transfer-agreement-respond-proposal.component.ts

```diff
@@ -38,7 +38,8 @@ import { WattIconComponent } from '@energinet-datahub/watt/icon';
 import { WattDatePipe } from '@energinet-datahub/watt/date';
 import { translations } from '@energinet-datahub/eo/translations';
 
-import { EoTransferAgreementProposal, EoTransfersService } from './eo-transfers.service';
+import { EoTransferAgreementsService } from './eo-transfer-agreements.service';
+import { TransferAgreementProposal } from './transfer-agreement.types';
```

Comment:
According to other naming convention, should this be called eo-transfer-agreements.types?